### PR TITLE
Add umis recipe.

### PR DESCRIPTION
**Pull request by roryk**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This is a set of useful tools for dealing with UMI and molecular
barcode tagged FASTQ files.

@bioconda/core

### CODE REVIEW

File: recipes/umis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package:
+  name: umis
+  version: '0.2.2a0'
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/umis/meta.yaml]: Build number should start at 0.

File: recipes/umis/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  skip: True # [osx]
```
Comment (bgruening) [inline recipes/umis/meta.yaml]: @roryk why have you added this step your first commit has passed, isn't it?

**Comment by roryk** [comment]
It originally failed on one of the OSX builds, but maybe it was a one off error. I reverted and repushed, if it builds I'll squash them out and force push.

**Comment by bgruening** [comment]
This looks good now, can you please start the build number at 0.
Thanks @roryk!

**Comment by roryk** [comment]
Thanks @bgruening -- fixed the build and squashed all of the commits together to clean the history for the merge.

**Comment by bgruening** [comment]
Awesome! Feel free to merge as soon as you see green :)

**Pull request merged**